ratis-server/src/test/java/org/apache/ratis/InstallSnapshotNotificationTests.java

```diff
@@ -332,6 +337,11 @@ private void testInstallSnapshotNotificationCount(CLUSTER cluster) throws Except
 
       // Make sure leader and followers are still up to date.
       for (RaftServer.Division follower : cluster.getFollowers()) {
+        long leaderIndex = leader.getRaftLog().getNextIndex();
+        // Give follower slightly time to catch up
+        RaftTestUtil.waitFor(
+                () -> leaderIndex == follower.getRaftLog().getNextIndex(),
+                300, 15000);
         Assert.assertEquals(
             leader.getRaftLog().getNextIndex(),
             follower.getRaftLog().getNextIndex());
```

Comment:
We may remove this Assert since we already have waitFor.

Reply:
Yes, I tought about that too, but I wanted to make absolutely certain that functionality of testcase is not changed accidentally. However, if you agree with my inner thoughts, then it is fine to remove. Fixed in the last commit.